<ui version="4.0" >
 <class>prog8051</class>
 <widget class="QDialog" name="prog8051" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Prog8051</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>SerialPort:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SerialPort" >
       <property name="text" >
        <string>/dev/ttyUSB0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Baudrate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Baudrate" >
       <item>
        <property name="text" >
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>4800</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Hex File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="HexFile" />
     </item>
     <item>
      <widget class="QPushButton" name="BrowseButton" >
       <property name="text" >
        <string>&amp;Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="Instructions" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Adjust Settings.......</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="ProgressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="ProgramButton" >
       <property name="text" >
        <string>&amp;Program</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton" >
       <property name="text" >
        <string>&amp;Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="QuitButton" >
       <property name="text" >
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
